src/app/api/api.py

```diff
@@ -344,16 +384,23 @@ async def filter_pending_collections(
     status_code=HTTP_200_OK,
     responses={HTTP_404_NOT_FOUND: {"description": "Collection not found"}},
     tags=[_TAG_COLLECTIONS],
-    description="""A list of missing entries for different types of materials belonging to the collection and
-    its subcollectionsspecified by 'node_id'.
-    Searches for materials with one of the following properties being empty or missing: """
-    + f"{', '.join([entry.value for entry in missing_attribute_filter])}.",
 )
 async def filter_materials_with_missing_attributes(
     *,
     node_id: uuid.UUID = Depends(node_ids_for_major_collections),
     missing_attr_filter: MissingAttributeFilter = Depends(materials_filter_params),
 ):
+    """
+    A list of missing entries for different types of materials belonging to the collection and its sub-collections
+    specified by 'node_id'.
+
+    Searches for materials with one of the following properties being empty or missing:
+    - properties.cm:title
+    - properties.cm:name
+    - properties.cclom:general_keyword
+    - properties.cclom:general_description
+    - properties.ccm:commonlicense_key
```

Comment:
There should be more properties, i.e., all properties belonging to materials as listed on line 258.

Reply:
done, I went through the individual lists and enums and added a lt more comments/fixmes here!